skiko/src/commonMain/kotlin/org/jetbrains/skia/impl/Native.kt

```diff
@@ -45,16 +45,18 @@ expect class InteropScope() {
     fun release()
 }
 
+typealias InteropScopeContext = InteropScope.(InteropPointer) -> Unit
```

Comment:
`InteropScopeContext` is somewhat misname, IMO, it's not context, but block of code, maybe `InteropScopeBlock` or `InteropScopeFunction`, interop context is created by functions like `withResult`.
Maybe readability wouldn't suffer much is no type alias was involved at all.

Reply:
I do agree with both statements - InteropScopeBlock is a better name and not having alias at all is also an option.
Choosing between this two options I decided to choose one that introduce less entities - so no alias at all. This change is pushed.